Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -378,6 +388,11 @@
 					for iMask=1:size(x.S.InputMasks,3)
 						if sum(SumList(iROI,iMask))~=0 % skip empty ROIs
 							x.S.InputMasks(:,iROI,iMask) = xASL_im_CreatePVEcROI(x,x.S.InputMasks(:,iROI,iMask), pGM_MNI, pWM_MNI);
+                        else
+                            numVoxels = sum(x.S.InputMasks(:,iROI,iMask));
+                            fprintf('\n');
+                            warning('%s\n', ['Current ROI ' namesROIlocal{iROI} ' only contains ' xASL_num2str(numVoxels) ' voxels, so this ROI will be skipped']);
+                            fprintf('%s\n\n', 'Consider reducing the minimal ROI volume setting x.S.MinimalROIVolume to also calculate this ROI');

```

Comment:
volume _by_ setting

calculate->evaluate. Or something like that.

Reply:
Done in 46c421bcd1aed8156cbe146a8cf708930de3d337